lms/djangoapps/commerce/utils.py

```diff
@@ -96,9 +96,9 @@ def payment_page_url(self):
         """ Return the URL for the checkout page.
 
         Example:
-            http://localhost:8002/basket/add/
+            http://localhost:1998
         """
-        return self.get_absolute_ecommerce_url(self.config.basket_checkout_page)
```

Comment:
Do we still need the `get_absolute_ecommerce_url` function? As far as I can see it returns the URL of the e-commerce legacy, does it still has usage?

I mean maybe we need to remove the function itself? Or it will be a part of LMS-checkout related task?

Reply:
this function also used for some other urls for example [get_order_dashboard_url](https://github.com/raccoongang/edx-platform/pull/2424/files/6fb92ccb745b1ca139b6a148db40a9a9627f38bc#diff-9d97b0582806ed85f7cd7b49ffc075dc2388097ca4b12a80b2c84a7c59d03feeR62)